.npmrc

```diff
@@ -0,0 +1,3 @@
+# Allow running `prepare` script from shire
```

Comment:
sure ?

Reply:
Option need but forgot to rename, here's without this `unsafe-perm`:

<img width="681" alt="Screenshot 2022-03-08 at 11 54 38" src="https://user-images.githubusercontent.com/43268759/157224755-c02dbd83-9d63-4ec4-9ffa-5ae6cc0efaba.png">